pkg/views/user/create/view.go

```diff
@@ -67,3 +72,30 @@ func CreateUserView(createView *CreateView) {
 		log.Fatal(err)
 	}
 }
+
+
+func IsValidPassword(password string) bool {
+	lengthPattern := `^.{8,128}$`
```

Comment:
Can you merge the regex into one?

Reply:
Combining the regex causing error. but, I optimized the PR better!